application/dash_application/views/preqc_wgs.py

```diff
@@ -29,33 +28,39 @@
     "second-sort",
     "colour-by",
     "shape-by",
+    "reads-per-start-point-slider",
+    "insert-mean-slider",
     "passed-filter-reads-slider",
 
     # Graphs
     "total-reads",
     "mean-insert",
+    "reads-per-start-point",
     "duplication",
     "purity",
     "ploidy",
     "unmapped-reads",
     "non-primary-reads",
     "on-target-reads",
 
+    "terminal-output",
     "data-table",
 ])
 
-BAMQC_COL = gsiqcetl.column.BamQcColumn
+BAMQC_COL = gsiqcetl.column.BamQc3Column
 ICHOR_COL = gsiqcetl.column.IchorCnaColumn
 PINERY_COL = pinery.column.SampleProvenanceColumn
 INSTRUMENT_COLS = pinery.column.InstrumentWithModelColumn
 RUN_COLS = pinery.column.RunsColumn
 
 special_cols = {
-    "Total Reads (Passed Filter)": "total reads passed filter",
-    "Unmapped Reads": "percent unmapped reads",
-    "Non-Primary Reads": "percent non-primary reads",
-    "On-target Reads": "percent on-target reads",
-    "Purity": "percent purity",
+    "Total Reads (Passed Filter)": "total_reads_pf",
+    "Unmapped Reads": "unmapped_reads_pct",
+    "Non-Primary Reads": "non_primary_reads_pct",
+    "On-target Reads": "on_target_reads_pct",
+    "Purity": "purity_pct",
+    "Project": "project",
+    "shape": "shape",
```

Comment:
Can this be reverted? I changed the column names because the actual strings in the BamQC dataframe column headers use "lowercase space words" instead of "snake_case". There's also no need to add the Project and shape columns here.

Reply:
Fixed.